src/pages/Index.less

```diff
@@ -0,0 +1,52 @@
+.index-page {
+    height: 100%;
+    min-height: 660px;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &__header,
+    &__description,
+    &__preview,
+    &__actions {
+        text-align: center;
+    }
+
+    &__header {
+        color: #ccc;
+        margin: 0 0 20px;
```

Comment:
Why not using Bootstrap defaults?

Reply:
why do?

Reply:
I understood we shouldn't take design considerations beyond the Bootstrap defaults. Otherwise, further design iterations will be way more difficult to achieve.